windtunnel/pre_processing.py

```diff
@@ -110,6 +110,13 @@ def compute_object_length(mesh: pv.PolyData):
 
 
 def save_mesh_obj(mesh, dest_object_path):
-    trimesh_mesh = trimesh.Trimesh(vertices=mesh.points,
-                                   faces=mesh.faces.reshape((-1, 4))[:, 1:])
+    faces = mesh.faces
+    if faces.shape[0] % 3 == 0:
```

Comment:
Please apply the DRY principle here, use the if just o detect the shape_vertices (either 3 or 4) but then the call to Trimesh is the same, passing the variable in the reshape.